projects/ccf-eui/src/app/shared/components/spatial-search-config-behavior/spatial-search-config-behavior.component.html

```diff
@@ -0,0 +1,3 @@
+<ccf-spatial-search-config (updateSex)="updateSex($event)" (updateOrgan)="updateOrgan($event)" (closeDialog)="close()"
```

Comment:
Pass sex as an input to prevent the behavior and pure component from getting out of sync